cmake.deps/cmake/GetBinaryDeps.cmake

```diff
@@ -10,17 +10,10 @@ function(GetBinaryDep)
     "INSTALL_COMMAND"
     ${ARGN})
 
-  if(NOT _gettool_TARGET OR NOT _gettool_INSTALL_COMMAND)
-    message(FATAL_ERROR "Must pass INSTALL_COMMAND and TARGET")
-  endif()
-
   string(TOUPPER "${_gettool_TARGET}_URL" URL_VARNAME)
   string(TOUPPER "${_gettool_TARGET}_SHA256" HASH_VARNAME)
   set(URL ${${URL_VARNAME}})
   set(HASH ${${HASH_VARNAME}})
-  if(NOT URL OR NOT HASH )
-    message(FATAL_ERROR "${URL_VARNAME} and ${HASH_VARNAME} must be set")
```

Comment:
Why isn't this needed/wanted?

Reply:
ExternalProject will emit the same warning anyway, so feels a bit unnecessary.